src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,173 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+import Link from "next/link";
+import { Icon } from "@iconify/react";
+
+export default function Navbar() {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const toggleMenu = () => setIsMenuOpen(!isMenuOpen);
+
+  const menuLinks = [
+    { name: "About", href: "/about" },
+    { name: "Fees", href: "/fees" },
+    { name: "Team", href: "/team" },
+    { name: "Testimonials", href: "/testimonials" },
+    { name: "FAQs", href: "/faqs" },
+  ];
+  const buttons = [{ name: "Book an Appointment", href: "/book-appointment" }];
+  return (
+    <nav
+      className={`${
+        isMenuOpen ? "bg-transparent shadow-none" : "bg-[#1E3432] shadow-lg"
```

Comment:
Can you use display none instead of bg transparent? then you don't need to handle different elements differently.